src/Parser.php

```diff
@@ -234,8 +234,12 @@ private function parseKeyValue($raw_line)
      *
      * @return array The .env contents
      */
-    public function getContent()
+    public function getContent($keyName = null)
     {
-        return $this->lines;
+		if ($keyName !== null) {
+			return (array_key_exists($keyName, $this->lines)) ? $this->lines[$keyName] : null;
+		} else {
+			return $this->lines;
```

Comment:
Looks good but maybe change this to
```
if (!is_null($keyName)) {
    return array_key_exists($keyName, $this->lines) ? $this->lines[$keyName] : null;
}

return $this->lines;
```

Reply:
Are ```is_null``` and ```$keyName !== null``` different?
Can you explain that?

According to the is_null function [comment](http://php.net/manual/en/function.is-null.php#117344), one guy said the ```is_null``` is faster than using ```=== null```.

Reply:
As @enygma says, using the ```!is_null``` or ```!== null``` doesn't matter in this code enhancement.
But the ```else``` condition should be removed to enhance the readability.

Reply:
@peter279k see https://github.com/m1/Env/pull/10#issuecomment-396332948 - it was to be consistent throughout the codebase rather than a performance related fix.

Reply:
You're right. It's consistent to use the ```is_null``` function.
This [line](https://github.com/m1/Env/blob/master/src/Parser.php#L171) also uses the ```is_string``` function.